showcase/examples/candlestick/candlestick-example.js

```diff
@@ -25,9 +25,10 @@ import {
   YAxis,
   LineSeries,
   makeWidthFlexible
-} from 'react-vis';
+} from 'index';
 import './candlestick.scss';
 import Candlestick from './candlestick';
+import './candlestick.scss';
```

Comment:
duplicated import, but can we actually keep this new one and add a newline to separate it from the js imports?